src/types/api.ts

```diff
@@ -35,6 +40,7 @@ export interface PullRequest {
   avatarUrl: string;
   status: PullRequestStatus;
   events: PullRequestEvent[];
+  lastComment: LastComment|null;
```

Comment:
Why has this been added to the API?

Reply:
This should be entirely on the server side, and rolled into the `hasNewActivity` boolean below.